app/decorators/proposal_decorator.rb

```diff
@@ -40,7 +40,7 @@ def display_status
   end
 
   def generate_status_message
-    if object.approvals.non_pending.empty?
+    if object.steps.non_pending.empty?
```

Comment:
would it make sense to add a method on proposal call `approvals` that returns approval steps, so we could still cal `object.approvals.non_pending.empty?` here?